lib/renderer.js

```diff
@@ -1134,6 +1134,27 @@
         
         return hasZeroBounds(this._getContentBounds(component.document.layers));
     };
+
+    /**
+     * checks to see if there is any complex mask applied to any of the layerGroup. 
+     * Complex Mask is defined as : 
+     * (1) Bitmap masks with extendWithWhite Property (Photoshop>Layers>Layer Mask>Hide Selection)
+     * (2) Vector masks or Bitmap Mask with zero bounds.(Layer Mask>Hide All or Vector Mask>Hide All)
+     * @private
+     * @param {!Component} component 
```

Comment:
There should also be an `@return` annotation. There doesn't seem to be one on the other 2 `_componentHas...` methods either, so can you update those also?